### ISSUE TITLE
mangashost.com (uBlock Origin)

### ISSUE THREAD

**Issue by leandroruel** [reek/anti-adblock-killer#3791]
please block www.mangashost.com

**Comment by uBlock-user** [comment]
force update uBlock Filters!

**Comment by leandroruel** [comment]
still appears some messages blocking some resources of the page, adblock detection still works, at least the full block don't appear anymore.

**Comment by okiehsch** [comment]
If you add 
`mangahosts.com##script:inject(setTimeout-defuser.js, testDisplay, 3000)`
are you still getting adblocking messages and if so can you provide a specific link?

**Comment by uBlock-user** [comment]
Yep the second one took care of it. They were running two scripts.

**Comment by sekedus** [comment]
try this one: [https://adblockfast.com](https://adblockfast.com)
